backend/.env.sample

```
@@ -2,3 +2,4 @@
 DATABASE_URL=postgres://permaplant:permaplant@127.0.0.1/permaplant
 BIND_ADDRESS_HOST=127.0.0.1
 BIND_ADDRESS_PORT=8080
+REMOTE_JWKS_URL=http://localhost:8081/realms/PermaplanT/protocol/openid-connect/certs
```

I think JWKS vs. JWK only leads to unnecessary typos, what about AUTH instead? Explanations what uid:100 could be, see #329:

```suggestion
REMOTE_AUTH_URL=uid;100
```

---

I think I had AUTH_URL previously, but changed it. The reason is that the URL has only to do with jwks (jsonwebtokens) and I believe REMOTE_AUTH_URL is in that sense less expressive as you would have to first read the code/doc to understand what AUTH means. For somebody that actually has to worry about changing something there JWKS is clearer in my opinion.

---

Ok, I created jwks.permaplant.net (@lukashartl)

---

> Ok, I created jwks.permaplant.net (@lukashartl)

I am currently setting up the LXC container for it.